includes/model/class-cached-actor.php

```diff
@@ -0,0 +1,99 @@
+<?php
```

Comment:
why having another actor? can't we use the already existing models?

Reply:
My assumption was that those models have way too much to override, given the way that they dynamically generate properties and instead I'm trying to rely on static cached Actor data. But maybe the Model could be responsible for its own fallback? Could we actually do `Actor::int_from_json( $cached_data )` and then bypass the `get_*` dynamic methods once those properties have been set?